projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -278,13 +279,14 @@ function clickOnConflictSolverLink(attribute: string, linkName: string): void {
  * @param attribute - Attribute name
  */
 export function isConflictLinkAttached(attribute: string): void {
-  cy.get('cx-configurator-attribute-header').within(() => {
-    cy.get(`#cx-configurator--attribute-msg--${attribute}`, {
+  cy.get('cx-configurator-attribute-header').as('conf');
+  cy.get('@conf')
+    .find(`#cx-configurator--attribute-msg--${attribute}`, {
       timeout: 10000,
-    }).within(() => {
+    })
+    .within(() => {
```

Comment:
What is the purpose of this refactor?

Reply:
Same refactor as above to handle 'within' command  Cy12 restriction, following workaround by migration guide, 'within + inner get' was replaced by 'get alias + find' 
```
 cy.get('cx-configurator-attribute-header').within(() => {
    cy.get(`#cx-configurator--attribute-msg--${attribute}`, {
      timeout: 10000,
    })
```
replaced by
```
 cy.get('cx-configurator-attribute-header').as('conf');
  cy.get('@conf')
    .find(`#cx-configurator--attribute-msg--${attribute}`, {
      timeout: 10000,
    })
```

The second 'within' (line 287) is not throwing any error, so stays as it is
```
.within(() => {
      cy.get('a.cx-action-link').wait(1000);
    });
```